File: algebra/sum_of_first_n_numbers/c#/SumOfFirstNNumbers.cs
namespace SumOfFirstNNumbers
{
    public class SumOFFirstNNumbers
    {
        static int SumOfFirstNNumbers(int n)
        {
            int sum = 0;
            for (int i = 1; i <= n; i++)
            {
                sum += i;
            }

            return sum;
        }
    }
}

File: algebra/sum_of_first_n_numbers/java/SumOfFirstNNumbers.java
public class SumOfFirstNNumbers {
    public static long sum(int n) {
        return (n * (n + 1)) / 2;
    }
public static void main(String args[]){
Scanner sc=new Scanner(System.in);
    System.out.println("\nEnter the value of n :");
    int n=sc.nextInt();
    System.out.println("Sum of first "+n+" numbers is "+sum(n));
}
}
